lms/templates/open_ended_problems/open_ended_flagged_problems.html

```diff
@@ -1,62 +0,0 @@
-<%! from django.utils.translation import ugettext as _ %>
-<%inherit file="/main.html" />
-<%block name="bodyclass">${course.css_class}</%block>
-<%namespace name='static' file='/static_content.html'/>
-
-<%block name="headextra">
-<%static:css group='style-course-vendor'/>
-<%static:css group='style-course'/>
-</%block>
-
-<%block name="pagetitle">${_("{course_number} Flagged Open Ended Problems").format(course_number=course.display_number_with_default)}</%block>
-
-<%include file="/courseware/course_navigation.html" args="active_page='open_ended_flagged_problems'" />
-
-<%block name="js_extra">
-    <%static:js group='open_ended'/>
```

Comment:
Unless you've included this elsewhere, see potential companion here: https://github.com/edx/edx-platform/pull/10846 .